x/stakeibc/keeper/update_redemption_rates_test.go

```diff
@@ -3,125 +3,111 @@ package keeper_test
 import (
 	// "fmt"
 
-	"fmt"
 	"math/rand"
 
 	sdkmath "cosmossdk.io/math"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	_ "github.com/stretchr/testify/suite"
 
+	minttypes "github.com/Stride-Labs/stride/v8/x/mint/types"
+
 	recordtypes "github.com/Stride-Labs/stride/v8/x/records/types"
 
 	stakeibctypes "github.com/Stride-Labs/stride/v8/x/stakeibc/types"
 )
 
-type UpdateRedemptionRatesTestCase struct {
-	hostZone   stakeibctypes.HostZone
-	allRecords []recordtypes.DepositRecord
+// TODO: Use array-of-test-cases setup for this redemption rate test
+// TODO [LSM]: Fix randomized test
```

Comment:
Recognizing how sensitive changes to the RR calc can be, can we hold off on merging this PR until all the TODO[LSM] tests are written and merged into this PR? Or is merging this PR blocking y'all from writing code for parts of the Stride-side LSM spec?

Reply:
those todos are already implemented in #725. We can merge #725 into this one first if you'd prefer

Reply:
#725 has been merged into this PR